src/plugins/sync-plugin.js

```diff
@@ -139,7 +138,7 @@ export const ySyncPlugin = (yXmlFragment, { colors = defaultColors, colorMapping
         update: () => {
           const pluginState = plugin.getState(view.state)
           if (pluginState.snapshot == null && pluginState.prevSnapshot == null) {
-            if (changedInitialContent || !fun.equalityDeep(view.state.doc.type.createAndFill().content.toJSON(), view.state.doc.content.toJSON())) {
+            if (changedInitialContent || view.state.doc.content.findDiffStart(view.state.doc.type.createAndFill().content) != null) {
```

Comment:
Nice catch! Thank you for the PR. Could you please change that to a strict in-equality check? `!==` instead of `!=`. 

Then I will publish a new release with your fix.

Reply:
updated